ci-operator/config/maistra/xns-informer/maistra-xns-informer-maistra-2.6.yaml

```diff
@@ -0,0 +1,45 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.6"
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: "1"
+      memory: 1Gi
+tests:
+- as: unit-2-6
```

Comment:
there's no need to put the version here. I literally meant just `unit`, `lint`, etc.